src/classes/Extend/ThemesDenylist.php

```diff
@@ -16,6 +16,11 @@ trait ThemesDenylist {
 	public function add_denylist_theme_notice() : void {
 		$current_theme = \wp_get_theme();
 
+		// Check for theme.
+		if ( ! in_array( $current_theme->get( 'Name' ), $this->themes_denylist ) ) {
+			return;
+		}
+
 		$message = sprintf(
 			esc_html__( '%1$s theme has been shown to have poor performance. We recommend switching to a different theme.', 'woocart-defaults' ),
```

Comment:
```suggestion
			esc_html__( '%1$s theme has been shown to have poor performance. We recommend switching to a different theme.', 'woocart-defaults' ),
```

Let's link `has been shown to have poor performance` to https://woocart.com/blog/fastest-woocommerce-theme